build.cake

```diff
@@ -242,11 +208,11 @@ Task("Test35")
 var testNetStandard20 = Task("TestNetStandard20")
     .Description("Tests the .NET Standard 2.0 version of the framework");
 
-foreach (var runtime in new[] { "netcoreapp3.1", "net5.0", "net5.0-windows" })
+foreach (var runtime in NetCoreTests)
 {
     var task = Task("TestNetStandard20 on " + runtime)
         .Description("Tests the .NET Standard 2.0 version of the framework on " + runtime)
-        .WithCriteria(IsRunningOnWindows() || runtime != "net5.0-windows")
+        .WithCriteria(IsRunningOnWindows() || !runtime.EndsWith("windows"))
```

Comment:
We no longer have any entries in `NetCoreTests` that ends with `windows`.

Reply:
This is the same as in main branch. It protects against someone updating `NUnitRuntimeFrameworks` in `Directory.Build.props`. We could remove the whole condition in both branches.